frontend/src/hooks/useWorkoutPlanPresetsApi.ts

```diff
@@ -1,42 +1,29 @@
 import { deleteItem, fetchData, sendData, updateItem } from "@/API/api";
-import { ICompleteWorkoutPlan, IWorkoutPlanPreset } from "@/interfaces/IWorkoutPlan";
+import { IWorkoutPlanPreset } from "@/interfaces/IWorkoutPlan";
 
-const WORKOUT_PLAN_ENDPOINT = "workoutPlansPresets/";
+const WORKOUT_PLAN_PRESETS_ENDPOINT = "presets/workoutPlans/";
 
-export const useWorkoutPlanPresetApi = () => {/* 
-  
-
```

Comment:
All this was stuff i deleted but notice how much space and comments there were. Before you open PRs go through the files you've worked on and make sure they are formatted properly and that you've deleted any unnecessary comments and console.logs etc....

Reply:
@Noam-Moskowitz Did you see this? If yes mark it as resolved or let me know and i will.